README.Rmd

```diff
@@ -11,46 +11,240 @@ knitr::opts_chunk$set(fig.path = "man/figures/README_", warning=FALSE, message=F
 
 [![Build Status](https://travis-ci.org/dynverse/dyngen.svg)](https://travis-ci.org/dynverse/dyngen) [![codecov](https://codecov.io/gh/dynverse/dyngen/branch/master/graph/badge.svg)](https://codecov.io/gh/dynverse/dyngen)
 
-A package to generating synthetic single-cell data starting from regulatory networks. The data is generated in several steps:
-![generation_overview](man/figures/README_generation_overview_v1.svg)
+A package for generating synthetic single-cell data starting from regulatory networks. The data is generated in several steps:
```

Comment:
Would be usefult o link to the exact kintetic model somewhere, e.g. the dynbenchmark sup note

Reply:
true, although this is already outdated by now, i think it should be included into the dyngen repo instead